src/app/Http/Controllers/API/PollController.php

```diff
@@ -171,12 +175,19 @@ public function results(Request $request, Poll $poll)
      * @urlParam user required User id for the vote
      * @bodyParam choice_id id required Choice id for the choosed choice
      *
-     * @param Request $request
+     * @param VoteRequest $request
      * @param Poll $poll
      * @param User $user
      */
-    public function vote(Request $request, Poll $poll, User $user)
+    public function vote(VoteRequest $vote_request, Poll $poll, User $user)
     {
-        //
+        $user->polls()->attach($poll, [
+            'choice_id' => $request->choice_id,
```

Comment:
`$request` is undefined, only work when changed to `$vote_request`

Reply:
mb

Reply:
fixed in a77e3cac6f5a711970fea6f4f8ed25d0dcaa8d1d

Reply:
All good